### ISSUE TITLE
Topnavigation, Mobile: shadown not shown on 0

### ISSUE THREAD

**Issue by martinheidegger** [consento-org/consento-website#69]
When the menu is closed, its okay/expected for the shadow to be hidden on mobile devices but when the menu is opened, the shadow should be visible, right now its not

<img src="https://user-images.githubusercontent.com/914122/74837151-2de46800-5364-11ea-9540-0c5c24144d48.png" width="200">

**Comment by Torgian** [comment]
Refer to [commit](https://github.com/consento-org/consento-website/commit/02706a97a3b2ce7838ce82d72860b8cc24b616d9)

I tried a few different methods on adding / removing the shadow based on element offsetTop, as well as trying to make the code a bit more compact. However, I ran into problems when invoking a Promise based on the css transition ending. I'd still like to explore this option in the future, but respect to our time-table, I decided to go for a more simple version. 

Tests run 4.0 for the following:

Shadow is added / removed when clicking the menu toggle button and the window is scrolled all the way to the top.
Shadow is _not_ removed when menu is opened, and the user scrolls all the way back to the top of the window. 
Shadow is _not_ removed if the window is scrolled down, and the menu is opened and/or closed. 

Again, I tried to think of a better way to implement this, but I don't want to spend too much time on this until after the other issues are fixed.
